packages/sdks/mitosis.config.cjs

```diff
@@ -864,6 +864,29 @@ const ANGULAR_MARK_SAFE_INNER_HTML = () => ({
   },
 });
 
+/**
+ * Angular allows DOM manipulation in `ngAfterViewInit` hook.
+ * This plugin moves the DOM code from `ngOnInit` to `ngAfterViewInit` hook.
+ */
+
+const ANGULAR_MOVE_DOM_MANIPULATION_CODE_TO_AFTERVIEWINIT = () => ({
+  code: {
+    post: (code,json) => {
+      if(json.name === 'BuilderVideo'){
+        const startIndex = code.indexOf('if (this.lazyLoad) {');
+        const endIndex = code.indexOf('this.lazyVideoObserver = oberver;')+34;
+        const codeToMove = code.slice(startIndex, endIndex+9);
+
+        code = code.slice(0,startIndex)+ code.slice(endIndex+9);
+
+        const ngAfterViewInitIndex = code.indexOf('this.setAttributes(this.elRef1?.nativeElement, this.node_2_source);')+67;
+        code = code.slice(0,ngAfterViewInitIndex+1)+ codeToMove + code.slice(ngAfterViewInitIndex+1);
+      }
+      return code;
+    },
+  },
+});
```

Comment:
These hardcoded indexes are too brittle and will quickly backfire.

I dug into our mitosis angular generator logic, and found that we have an undocumented way to move hooks from one place to another:

```suggestion
const ANGULAR_MOVE_DOM_MANIPULATION_CODE_TO_AFTERVIEWINIT = () => ({
  json: {
    post: (json) => {
      if (json.name === 'BuilderVideo') {
        json.compileContext = {
          angular: {
            hooks: {
              ngAfterViewInit: json.hooks.onMount[0],
            },
          },
        };

        delete json.hooks.onMount;
      }
    },
  },
});
```

This should be much cleaner and still accomplish the same task